client/features/search/Search.jsx

```diff
@@ -64,8 +65,11 @@ class Search extends React.Component {
             className='search-button'>Search
           </button>
         </div>
-        <TempDisplay1 data={this.state.searchDisplay}/>
-        <TempDisplay2 data={this.state.searchDisplay}/>
+        {this.state.searchDisplay.length ?
+          <MediaTileCarousel tempData={this.state.searchDisplay} label={ 'Search Results' } />
```

Comment:
Do you want to show search results in a carousel? Carousels are design requirements for the watch lists on Home, but here you can just fill the screen.

Reply:
resolved